sdk/storage/azure-storage-blob/azure/storage/blob/_deserialize.py

```diff
@@ -83,13 +88,21 @@ def deserialize_ors_policies(policy_dictionary):
     return result_list
 
 
-def deserialize_blob_stream(response, obj, headers):
```

Comment:
I can't leave a comment on line 74:
`parsed_result = {}`

But any help with this one 😄 
azure\storage\blob\_deserialize.py:74: error: Need type annotation for "parsed_result" (hint: "parsed_result: Dict[<type>, <type>] = ...")  [var-annotated]

Reply:
Should be Dict[str, List[ORP]]